src/functions/apiFunctions.tsx

```diff
@@ -1,5 +1,42 @@
-import { ArtistData } from "@/interfaces/artistInterface";
+import { GenreData } from "@/interfaces/genreInterface";
 import { ResponseData } from "@/interfaces/songInterface";
+import { ArtistData } from "@/interfaces/artistInterface";
+
+export function tallyArtist(data: ResponseData): Map<string, number> {
+  let artistCounts = new Map<string, number>();
```

Comment:
`artistCounts` should be a const

Reply:
genreCounts and artistCounts both changed to a const